### DISCUSSION TITLE
Add support for LlamaIndex

### DISCUSSION THREAD

**Discussion by schkovich** [cloudflare/workerd#1962] [Python Packages]
[LlamaIndex](https://pypi.org/project/llama-index/) is designed for building search and retrieval applications. It provides a simple interface for querying LLMs and retrieving relevant documents. Langchain and Llama Index are NOT mutually exclusive and could be used together to achieve the desired outcome.

**Comment by rawwerks** [comment]
linking these: https://github.com/cloudflare/workerd/discussions/2313
